ambry-replication/src/main/java/com/github/ambry/replication/ReplicationManager.java

```diff
@@ -269,24 +269,31 @@ public void onPartitionBecomeBootstrapFromOffline(String partitionName) {
 
     @Override
     public void onPartitionBecomeStandbyFromBootstrap(String partitionName) {
+      logger.info("Partition state change notification from Bootstrap to Standby received for partition {}",
+          partitionName);
```

Comment:
Since we already print this in Base class, we may see duplicate prints of similar messages on bootstrap to standby transition.

Reply:
its not printed in base class PartitionStateChangeListener